File: backend/src/main/java/pl/edu/agh/utp/records/request/TransactionRequest.java
package pl.edu.agh.utp.records.request;

import java.util.List;
import java.util.UUID;

public record TransactionRequest(
    String description,
    String date,
    String category,
    double amount,
    UUID paymentUserId,
    List<UUID> debtsUserIds) {}

File: frontend/app/src/main/java/pl/edu/agh/utp/model/transaction/TransactionRequest.kt
package pl.edu.agh.utp.model.transaction

import java.util.UUID

data class TransactionRequest(
    val description: String,
    val date: String,
    val category: String,
    val amount: Double,
    val paymentUserId: UUID,
    val debtsUserIds: List<UUID>
)
